index.js

```diff
@@ -106,6 +118,11 @@
     return f(this.value);
   };
 
+  //  Just#concat :: m a -> m a
+  Just.prototype.concat = function(maybe) {
+    return maybe instanceof Just ? Just(this.value.concat(maybe.value)) : this;
+  };
+
```

Comment:
> How do you know that you can concat the `value` of this `Maybe` with the value of that `Maybe`?

We don't, but according to the [spec][1]…

>     s.concat(b)
>
> 1.  `b` must be a value of the same Semigroup
>
>     1.  If `b` is not the same semigroup, behaviour of `concat` is unspecified.


[1]: https://github.com/fantasyland/fantasy-land